src/gui/history/qgshistoryentry.cpp

```diff
@@ -0,0 +1,37 @@
+/***************************************************************************
+                            qgshistoryentry.cpp
+                            -------------------------
+    begin                : April 2019
```

Comment:
Where you keeping this in store since 2019? :wink:

Reply:
I split it out from another file with that date... it seems right to keep the original :)